users/models.py

```diff
@@ -32,11 +32,17 @@ class TeachingType(models.TextChoices):
         default=TeachingType.TEACHING_PROF,
     )
     is_peng = models.BooleanField(default=False)
+    is_form_submitted = models.BooleanField(default=False)
 
     class Meta:
         managed = True  #auto creates tables
         db_table = 'appuser'
 
+# Use Django signals to delete User instance when AppUser is deleted. Based on: https://stackoverflow.com/a/12754229
+@receiver(post_delete, sender=AppUser)
+def delete_code_constraint_with_question(sender, instance, **kwargs):
```

Comment:
Is this a desired class method name? I think it is just a bit confusing to me

Reply:
Nice catch! I'll fix that

Reply:
I updated the name

Reply:
Looks good thanks!